elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -82,12 +83,14 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, $backwards-compatibility);
 
   @at-root #{&}(:focus) {
     border-bottom-color: pfe-local(BorderColor--hover);
+      background-color: pfe-local(BackgroundColor);
```

Comment:
Is it just me or does the spacing look different than the border-bottom-color above it?

Reply:
thanks, fixed